internal/publish/preflight_checks.go

```diff
@@ -17,7 +16,7 @@ type checkConfigurationSuccessData struct{}
 
 var errTypeChanged = errors.New("configuration type cannot be changed once deployed; create a new destination to use a different type")
 
-func (p *defaultPublisher) checkConfiguration(client connect.APIClient) error {
+func (p *defaultPublisher) preFlightChecks(client connect.APIClient) error {
```

Comment:
the functionality started out just checking the configuration, but we had always referred to it as the preflight checks. This renaming should help remove some of the confusion we've recently had.